src/main/java/httpserver/HttpResponse.java

```diff
@@ -0,0 +1,10 @@
+package httpserver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HttpResponse {
+    public String responseLine = "HTTP/1.1 ";
+    public String body = "";
+    public List<String> headers = new ArrayList<String>();
+}
```

Comment:
I'd like to add a new constraint.

Classes cannot have public instance variables. (This doesn't apply to test doubles)

Reply:
Sounds good.  Will refactor.

Reply:
What do you think of just returning a Map object instead of a class?

Reply:
The only problem with a map is that the data type of key value pairs are set

Reply:
That is, all values have the same type.

Reply:
Yeah, I am not sure a map would fit very well because of the nature of the data. The other downside to a Map is that you forgo the opportunity to create your API, and can only use what Map defines for you.